src/recoilState/vaults.ts

```diff
@@ -16,14 +16,19 @@ export const rVaults = atom({
   ],
 });
 
-export const useVaults = () => useRecoilValue(rVaults);
+export const useVaults = (orgId: number) => {
+  return useRecoilValue(rVaults)[orgId] || [];
+};
 
 export const useFakeVaultApi = () => {
   const addVault = useRecoilCallback(
     ({ snapshot, set }) =>
-      async (v: IVault) => {
+      async (org: number, v: IVault) => {
```

Comment:
nit, I like variable names to like `orgId` because to me org should be the org object